extension/webviews/process-editor/src/ivy-diagram-widget.ts

```diff
@@ -36,15 +39,33 @@ export abstract class IvyGLSPDiagramWidget extends GLSPDiagramWidget {
     this.actionDispatcher.dispatch(EnableToolPaletteAction.create());
     this.actionDispatcher.dispatch(EnableViewportAction.create());
 
-    this.messenger.onRequest(InitializeServerRequest, server => this.initServer(server));
+    this.messenger.onRequest(InitializeConnectionRequest, () => this.initConnection());
     this.messenger.sendNotification(WebviewReadyNotification, HOST_EXTENSION);
   }
 
-  private initServer(server: string) {
+  private initConnection() {
     if (this.actionDispatcher instanceof GLSPActionDispatcher) {
-      this.actionDispatcher
-        .onceModelInitialized()
-        .finally(() => this.actionDispatcher.dispatch(EnableInscriptionAction.create({ connection: { server } })));
+      this.actionDispatcher.onceModelInitialized().finally(() => {
+        const ivyScript = {
```

Comment:
Could you elaborate on the two MessageConnections that you are basically creating here? Once the model is ready, we create those connections to the messenger and expose them through the `EnableInscriptionAction`, however how is that used and why do we need to use WebSocketMessageReader and WebSocketMessageWriter as we only use it as a wrapper for the webview messenger?

Btw, you could probably also call `createWebSocketConnection(IWebSocket)` for a shorthand and "re-use" the created IWebSocket you create per type.

Reply:
Thank you for the review. As mentioned before, you should now be able to build and run the extension.

One Connection is used to to start the MonacoLangueClient, see https://github.com/axonivy/inscription-client/blob/12c19bd9801c8dee7996f561fa9d8b4da7e32e91/packages/core/src/ivy-script-client.ts#L12C30-L12C30

and the other one to start the InscriptionClientJsonRpc, see https://github.com/axonivy/inscription-client/blob/12c19bd9801c8dee7996f561fa9d8b4da7e32e91/packages/core/src/inscription-client-jsonrpc.ts#L84